pyfeatures/app/calc.py

```diff
@@ -43,6 +43,43 @@
 from pyfeatures.schema import Signatures as out_schema
 
 
+def get_image_size(fin):
+    """
+    There doesn't seem to be a nicer way to get the dimensions
+    """
```

Comment:
Yes, there is. You have to get the `shape` attribute from the data block and the `dimension_order` one from the main record. I can do this, should be straightforward.